config/tracks/pc_jingsu.ini

```diff
@@ -1,8 +1,40 @@
+[ABOUT]
+AUTHOR = CrisT86, Mitch9
+VERSION = 1
+DATE_RELEASE = 09/03/21
+LIGHTS_COUNT = ~
+NOTES = 
+ 
 [LIGHTING]
 LIT_MULT = 1
+SPECULAR_MULT = 1
+BOUNCED_LIGHT_MULT = 1
+CAR_LIGHTS_LIT_MULT = 1
 
+[INCLUDE]
+INCLUDE = common\conditions.ini, common\materials_base.ini, common\materials_track
+
+[LIGHT_POLLUTION]
+RELATIVE_POSITION=0,0,0
+DENSITY=0.5
+RADIUS_KM=4
+COLOR=1, 0.8, 0.4
+CONDITION=NIGHT_SHARP
+
+[BOUNCED_LIGHT]
+SUPPORTED = 1
+SURFACE_MATERIALS = road-new, Concrete_side, curb_new1, asph-pitlane?, grass?, Sand-16, kantai002_2
+
```

Comment:
You added surface materials but no object or occluding materials making the entire lightmap incoherent

Reply:
Done, yeah it works better now